content/en/webassembly.md

```diff
@@ -0,0 +1,18 @@
+---
+title: WebAssembly
+status: Feedback Appreciated
+category: Concept
+---
```

Comment:
```suggestion
category: Concept
tags: ["Application", "", ""]
---
```
We need to add tags according to 
https://glossary.cncf.io/style-guide/#tags

Considering Wasm is mostly related with application and has a similar position with the [Container](https://glossary.cncf.io/container), `Application` can be its tag. (In case of Container, tag1: Application, tag2: Fundamental, but Wasm is premature to say it is a `Fundamental` term.)

Reply:
+1 Application as its tag